pipeline/scripts/setup-emulator.sh

```diff
@@ -0,0 +1,25 @@
+#!/usr/bin/env bash
+emulator_config="system-images;android-$ANDROID_API_LEVEL;google_apis;x86_64"
+emulator_name="emulator_api_$ANDROID_API_LEVEL"
+
+# Install AVD files
+echo "y" | $ANDROID_HOME/tools/bin/sdkmanager --install "$emulator_config"
+# Create emulator
+echo "no" | $ANDROID_HOME/tools/bin/avdmanager create avd -n "$emulator_name" -k "$emulator_config" --force
+
+$ANDROID_HOME/emulator/emulator -list-avds
+
+echo "Starting emulator"
+
+# Start emulator in background
+nohup $ANDROID_HOME/emulator/emulator -avd "$emulator_name" -no-snapshot -no-boot-anim > /dev/null 2>&1 &
+$ANDROID_HOME/platform-tools/adb wait-for-device shell 'while [[ -z $(getprop sys.boot_completed | tr -d '\r') ]]; do sleep 1; done; input keyevent 82'
```

Comment:
It would be nice for someone watching a build in progress to echo something indicating that a wait is happening/what it's waiting for before each of the sleep-like lines

Reply:
I added more echos to the script. During the while loop, I just echo a . to create a ...... loading effect, since any words would be repeated a lot. Let me know if you have any suggested improvements!